Source/WebCore/rendering/svg/RenderSVGRoot.cpp

```diff
@@ -549,7 +549,7 @@ void RenderSVGRoot::mapLocalToContainer(const RenderLayerModelObject* repaintCon
     // For getCTM/getScreenCTM computations the result must be independent of the page zoom factor.
     // To compute these matrices within a non-SVG context (e.g. SVG embedded in HTML -- inline SVG)
     // the scaling needs to be removed from the CSS transform state.
-    TransformState transformStateAboveSVGFragment(settings().css3DTransformInteroperabilityEnabled(), transformState.direction(), transformState.mappedPoint());
+    TransformState transformStateAboveSVGFragment(true, transformState.direction(), transformState.mappedPoint());
```

Comment:
I think the only caller of the `TransformState` constructor that doesn't use the pref currently is just a mistake, so we could/should remove this param.

As with the above, doing so in a separate step is fine :)